standard.md

```diff
@@ -0,0 +1,114 @@
+
+- **diagnostic_path** : `str`. Path to the diagnostic executable. It may look redundant, but this way the file contains all the information required to run the diagnostic.
+- **input_files**: `list`. List of absolute paths to the metadata files describing the data to be used by the diagnostic. The format of this files is defined later.
+- **tool**: `str`. Name of the tool used to prepare the data for the diagnostic.
+- **version**: `str` Version of the tool used to prepare the data for the diagnostic.
+- **run_dir**: `str`. Path to the folder to use as current path for diagnostic execution. Must be writable by the diagnostic.
+- **data_dir**: `str`. Path to the folder to store the  diagnostic's generated data on. Must be writable by the diagnostic.
+- **plot_dir**: `str`. Path to the folder to store the  diagnostic's generated figures on. Must be writable by the diagnostic.
+
+It is suggested to provide different folders for `run_dir`, `data_dir` and `plot_dir`, as it will be easier for the users to interact with the results, but the same folder can be used.
+
+### Reserved options
+
+The options listed in this file are not required for the diagnostics to implement, but if they do they must follow the following guidelines:
+
+- **write_plots**: `boolean`, default value is `true`. A flag to control if the diagnostic generates the associated figures with the results or not.
+- **write_data**: `boolean`, default value is `true`. A flag to control if the diagnostic generates the associated data files with the results or not.
+- **log_level**: `str`, default values is `info`. Sets the granularity of the diagnostic logs and console output. If used, diagnostic must accept at least `error` `warning`, `info` and `debug`
+- **auxiliary_data_dir** : `str`. Path to the auxiliary directory where the diagnostic may find further input data.
+- **max_proc_number** : `integer`, default value is **1**. The maximum number of processors that the tool allows the diagnostic to request
+
+
+## The data definition file
+
+The data definition file must be a yaml file containing a dictionary of dictionaries. This first dictionary will have as keys the path to the data files and as values the dictionary containg the metadata of each file.
+
+There is no limit to the number of file that can be provided on a single file, although it is suggested
+
+### Required options
+
+- **alias**: `str`. Unique name for the dataset. Any (`alias`, `variable`) pair must be unique
+- **filename**: `str`. Absoulute path to the data file.
+- **variable**: `str`. Variable name to be used by the diagnostic. It does not have to match any of the CF conventions names in the file
+
+### Reserved options
+
+#### Related to the dataset
+
+- **project**: `str`. Name of the project at which this dataset belongs. It can be a real project, like `CMIP5`, `CMIP6` or a tool defined agrupation of datasets like `Observations` or `Reanalysis`
```

Comment:
```suggestion
- **project**: `str`. Name of the project to which this dataset belongs. It can be a real project, like `CMIP5`, `CMIP6` or a tool defined grouping of datasets like `Observations` or `Reanalysis`
```